core/src/main/java/org/hibernate/ogm/dialect/spi/GridDialect.java

```diff
@@ -8,6 +8,7 @@
 
 import org.hibernate.LockMode;
 import org.hibernate.dialect.lock.LockingStrategy;
+import org.hibernate.ogm.index.OgmIndexSpec;
```

Comment:
It has been removed in my branch as the createIndex method has been moved to the SchemaDefiner.